### ISSUE TITLE
Add install option to Settings

### ISSUE THREAD

**Issue by CinemaVision** [CinemaVision/service.cinemavision#7]
Add install link for CinemaVision Service Add-on to Settings > General > Extras.

**Comment by ruuk** [comment]
Yeah, thought about that but I want the add-on in the repo first because it's easier to test this feature then. So in a follow up version of the add-on.

**Comment by CinemaVision** [comment]
Fair enough!
